TOOLS/ce/src/main/scala/org.opalj.ce/ConfigObject.scala

```diff
@@ -52,20 +58,14 @@ case class ConfigObject(var entries: mutable.Map[String, ConfigNode], comment: D
             }
         } else {
             for ((key, node) <- entries) {
-                content += node.toHTML(key, HTMLHeadline, HTMLContent, sorted, maximumHeadlinePreviewLength) + "\n"
+                content += s"${node.toHTML(key, HTMLHeadline, HTMLContent, sorted, maximumHeadlinePreviewLength)} \n"
```

Comment:
This call to another `toHTML` that will create an internal StringBuilder, then call `toString` on it in the end just to be added into another StringBuilder here (+ string interpolation AFAIK also uses a StringBuilder internally) suggests to me that it might be better to have the method be `: Unit` and take a StringBuilder as a parameter to which the HTML is then added

Reply:
I reworked the entire export process to only use one String builder.
I also added additional operations that removed some helper variables within the export process, making the source code run more linear with each line of the HTML file